godtools/App/Features/Dashboard/Presentation/Tools/Models/ToolFilterSelection.swift

```diff
@@ -10,6 +10,6 @@ import Foundation
 
 struct ToolFilterSelection {
```

Comment:
Hey @rachaelblue looks like this file can be removed?  I don't think it's in use anymore.